services/ui-src/src/verbiage/pages/accordion.ts

```diff
@@ -122,40 +125,42 @@ export default {
-            "aria-label": "42 CFR § 438.8(l) (link opens in new tab)",
-          },
-        },
-        {
-          type: "text",
-          as: "span",
-          content:
-            ' a state may exclude a plan that is newly contracted with the state from this reporting for the first year of the plan\'s operation. These "new experience" plans must report MLRs during the next MLR reporting year in which the plan is in business with the state, even if the first year was not a full 12 months.</p>',
+          type: "p",
+          children: [
+            {
+              type: "html",
+              content:
+                "<p>States must provide summary MLR report data for each managed care plan. These reports are based on the plans' annual MLR reports to the state under ",
+            },
+            {
+              type: "externalLink",
+              content: "42 CFR § 438.8(k)",
+              props: {
+                href: "https://www.ecfr.gov/current/title-42/chapter-IV/subchapter-C/part-438/subpart-A/section-438.8#p-438.8(k)",
+                target: "_blank",
+                "aria-label": "42 CFR § 438.8(k) (link opens in new tab)",
+              },
+            },
+            {
+              type: "html",
+              content:
+                ". States have the option of reporting these data for each managed care program, statewide, or at another level of aggregation (e.g., eligibility groups). Managed care program is defined by a specified set of benefits and eligibility criteria that are articulated in a contract between the state and managed care plans. <b>In general, MLR data should not be aggregated across programs; however, if a managed care plan participates in more than one program with the state—the state can report the plan's MLRs for each program separately or combine the plan's results across managed care programs.</b> If a state combines the plan's reporting across programs, the report must use a consistent MLR reporting year.</p> <br/>States must report MLRs for both credible and non-credible MCOs, PIHPs, and PAHPs. There is no exception for MLR reporting for non-credible managed care plans. Under ",
+            },
+            {
+              type: "externalLink",
+              content: "42 CFR § 438.8(l)",
+              props: {
+                href: "https://www.ecfr.gov/current/title-42/chapter-IV/subchapter-C/part-438/subpart-A/section-438.8#p-438.8(l)",
+                target: "_blank",
+                "aria-label": "42 CFR § 438.8(l) (link opens in new tab)",
+              },
+            },
+            {
+              type: "html",
+              content:
+                " a state may exclude a plan that is newly contracted with the state from this reporting for the first year of the plan's operation. These “new experience” plans must report MLRs during the next MLR reporting year in which the plan is in business with the state, even if the first year was not a full 12 months.</p>",
```

Comment:
what is the close `</p>` tag for here? serving as a break?

Reply:
I'm not sure, I think I just missed it 😥 just removed it, thank you!!